ert_gui/tools/file/file_dialog.py

```diff
@@ -59,10 +60,24 @@ def _stop_thread(self):
         self._thread.quit()
         self._thread.wait()
 
-    def _init_layout(self):
-        self.setMinimumWidth(600)
-        self.setMinimumHeight(400)
+    def _calculate_desired_width(self):
```

Comment:
In the last line of this function you describe what is "desired" by a temporary variable. I would suggest renaming the function to `_calculated_font_based_width` and then a direct `return` at the end (no temp variable called `font_based_with`)

Reply:
done, similar renaming also applied to corresponding height function.